src/Host/Client/Impl/BrokerServices/WebServer.cs

```diff
@@ -31,18 +31,17 @@ public class WebServer {
         public string RemoteHost { get; }
         public int RemotePort { get; }
 
-        private WebServer(string remoteHostIp, int remotePort, string baseAddress, string name, ICoreServices services, IConsole console) {
+        private WebServer(string remoteHostIp, int remotePort, string baseAddress, string name, ICoreShell coreShell, IConsole console) {
```

Comment:
Same thing here. Generally, I'd prefer to use `ICoreShell` only where it is required.